streamingpro-mlsql/src/main/java/streaming/dsl/mmlib/algs/SQLCacheExt.scala

```diff
@@ -40,9 +40,10 @@ class SQLCacheExt(override val uid: String) extends SQLAlg with WowParams {
       "cache"
     }
 
-    val _lifeTime = CacheLifeTime.withName(params.get(lifeTime.name).getOrElse {
-      CacheLifeTime.SCRIPT.toString
-    })
+    var _lifeTime = CacheLifeTime.SCRIPT
```

Comment:
If you want to validate the liftTime, if it's invalid,  I think the best way is to throw exception instead doing nothing.